app/src/main/java/org/android/go/sopt/presentation/home/adapter/MyAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package org.android.go.sopt.presentation.home.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import org.android.go.sopt.data.remote.model.ResponseFollowerDto
+import org.android.go.sopt.databinding.ItemReqresFollowerBinding
+
+class MyAdapter(context: Context) : ListAdapter<ResponseFollowerDto.Data, MyAdapter.MyViewHolder>(
+    diffUtil
+) {
+    private val inflater by lazy { LayoutInflater.from(context) }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MyViewHolder {
+        val binding: ItemReqresFollowerBinding = ItemReqresFollowerBinding.inflate(inflater, parent, false)
+        return MyViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: MyViewHolder, position: Int) {
+        holder.onBind(currentList[position])
+    }
+
+    class MyViewHolder(private val binding: ItemReqresFollowerBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun onBind(item: ResponseFollowerDto.Data) {
+            binding.tvName.text = item.first_name
+            binding.tvEmail.text = item.email
+            Glide.with(binding.root)
+                .load(item.avatar)
+                .circleCrop()
+                .into(binding.ivImage)
+        }
+    }
+
+    companion object {
+        val diffUtil = object : DiffUtil.ItemCallback<ResponseFollowerDto.Data>() {
+            override fun areItemsTheSame(oldItem: ResponseFollowerDto.Data, newItem: ResponseFollowerDto.Data): Boolean {
+                return oldItem.id == newItem.id
```

Comment:
```suggestion
                return oldItem === newItem
```
@gaeun5744 님 말씀대로 쓰면 저렇게 쓸 수 있긴 하겠네요. 근데 아마 areItemsTheSame이 true를 반환하면 areContentsTheSame까지 체크해보는 로직으로 알고 있는데, 만약 ===로 비교하게 된다면 서버에서 새로운 데이터를 받아왔을 때 내용물은 완전히 동일하기 때문에 갱신할 필요가 없는 데이터에 대해서, 주소값이 다르기 때문에 갱신을 하게 된다는 우려도 생길 것 같습니다. 잘 고민해서 사용해보시면 될 것 같아요.

Reply:
아 이런 단점은 생각지도 못했네요!! 저도 배우고 갑니다~